sewer/config.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Created by @menduo @ 2018/7/25
+
```

Comment:
This is nice

Reply:
These code were generated by my IDE, I can remove them, or you can remove them too.